src/binaryen_stubs_expressions.c

```diff
@@ -103,9 +103,11 @@ caml_binaryen_call(value _module, value _name, value _params, value _retty) {
 }
 
 CAMLprim value
-caml_binaryen_call_indirect(value _module, value _target, value _params, value _paramsty, value _retty) {
-  CAMLparam5(_module, _target, _params, _paramsty, _retty);
+caml_binaryen_call_indirect(value _module, value _table, value _target, value _params, value _paramsty, value _retty) {
+  CAMLparam5(_module, _table, _target, _params, _paramsty);
+  CAMLxparam1(_retty);
```

Comment:
what's `xparam1`?

Reply:
The macros for `CAMLparam` only go up to 5, so if you've got more you need to use the "extra" params.